PWGGA/GammaConv/macros/AddTask_TRDEfficiency.C

```diff
@@ -38,7 +38,61 @@ AliAnalysisTRDEfficiency* AddTask_TRDEfficiency(
 
     // ================== GetInputEventHandler =============================
     AliVEventHandler *inputHandler=mgr->GetInputEventHandler();
-    
+    cout << "$$$$$$$$$ part of the task $$$$$$$$" << endl;
+    //=========  Set Cutnumber for V0Reader ================================
+    TString cutnumberPhoton = "00200009227300008250400000";
```

Comment:
the cutnumber should not be hardcoded, as they may vary for different data/MC sets